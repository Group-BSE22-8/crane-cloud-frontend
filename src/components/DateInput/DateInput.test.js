/* eslint-disable no-undef */
import React from "react";
import { shallow } from "enzyme";
import DateInput from ".";
const DateInputProps = {
    formatString: jest.fn(),
    createTimestamp: jest.fn(),
    getDate: jest.fn(),
    getTime: jest.fn(),
    trimMonthName: jest.fn(),
};
describe('Date input component', () => {
  const DateInputWrapper = shallow(<DateInput {...DateInputProps} />);
  it("date input  should match snapshot", () => {
    expect(DateInputWrapper).toMatchSnapshot();
  });
  it("Date input component should match the snapshot", () => {
    DateInputWrapper.setProps(DateInputProps);
    expect(DateInputWrapper).toBeDefined();
    expect(DateInputWrapper).toHaveLength(1);
  });
});
