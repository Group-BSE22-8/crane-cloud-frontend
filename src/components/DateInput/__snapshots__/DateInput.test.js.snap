// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Date input component date input  should match snapshot 1`] = `ShallowWrapper {}`;
